roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -23,7 +23,7 @@
   when: openshift.common.is_containerized | bool and not openshift.common.is_master_system_container | bool
 
 # workaround for missing systemd unit files
-- name: Create the systemd unit files
+- name: "Create the {{ openshift.common.service_type }} systemd unit file"
```

Comment:
Although vars in task names are cool and all, it makes it a bit trickier to find that troubling task in the codebase when you are looking at a task failure in the log.  No task name matches what you are looking for so head scratching ensues.  It may just be my way going about finding stuff.  Some name is better than having no name.